inc/blocks/ab-test/components/variant-panel.js

```diff
@@ -18,12 +45,17 @@ const { __ } = wp.i18n;
  * @param {React.ReactNode} props.placeholder Placeholder text for the variant title while loading.
  * @returns {React.ReactNode} Variant settings panel.
  */
-const VariantPanel = ( { title, variant, variants, onMouseDown } ) => {
+const VariantPanel = ( { index, blockId, postId, title, variant, variants, onMouseDown } ) => {
 	const { updateBlockAttributes } = useDispatch( 'core/block-editor' );
 
 	const defaultPercentage = 100 / variants.length;
 	const minPercentage = variants.length > 1 ? 0 : 100;
 
+	// Get A/B Test Block Id
+	const ab_post = useSelect( select => {
```

Comment:
I know in some of the code we have some snake case naming conventions - generally as a rule I always try to keep my code style consistent with the existing code in the file, so this variable name would become `abPost` for example.